### DISCUSSION TITLE
Absurd performance issues

### DISCUSSION THREAD

**Discussion by dries-dekeersgieter** [statamic/cms#9465] [Troubleshooting]
Hello,

I'm running into some absurd performance issues in my Statamic project.
The site is quite basic, except for the fact that it is a multi-site with 8 languages. (Important: i see no performance difference between a single site and multi site)
The site contains a total of about 100 entries.
When I display an empty antlers view, the render time is about 2.38s with a memory usage of about 85MB.

When I display a view with some content (navigation, collection with about 10 entries, no glide tags), the render time is about 6-9 seconds.
My local dev server runs on an Intel i9-12900KS, with 64GB RAM and SSD.
Running the server on Homestead and my Windows host, gives the same results.

I know cache strategies are an important method to increase site performance. But these slow page loads are just very noticeable during development and cold cache page loads on production. (half cache strategy). 

I really really hope i can get some clear insights and solutions on this matter.
I love using Statamic and i've invested a lot of time learning this ecosystem.
But these performance headaches keep popping up in most of my Statamic projects, which make me doubt my use of Statamic in the future.

Environment
Application Name: xxx
Laravel Version: 10.43.0
PHP Version: 8.3.1
Composer Version: 2.6.6
Environment: local
Debug Mode: ENABLED
URL: xxx
Maintenance Mode: OFF

Cache
Config: CACHED
Events: NOT CACHED
Routes: CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: statamic
Database: mysql
Logs: stack / single
Mail: smtp
Queue: sync
Session: file

Statamic
Addons: 5
Antlers: runtime
Stache Watcher: Enabled
Static Caching: Disabled
Version: 4.46.0 PRO

**Comment by jasonvarga** [comment]
We got your support ticket. No need to cross post. We'll help you over there.

**Comment by fabus961** [comment]
Same here, the performance is not as good as promised. I'm rather disappointed...
